var/spack/repos/builtin/packages/r-aneufinder/package.py

```diff
@@ -33,6 +40,10 @@ class RAneufinder(RPackage):
     depends_on('r-ggplot2', type=('build', 'run'))
```

Comment:
It doesn't look like `r-biocgenerics` is a dependency.

Reply:
Ping

Reply:
fixed